pkg/mkimage-raw-efi/install

```diff
@@ -298,15 +298,7 @@ eve_flavor=$(cat /root/etc/eve-hv-type)
 if [ "$eve_flavor" = "kubevirt" ]; then
    INSTALL_CLUSTERED_STORAGE=true
    INSTALL_ZFS=true
-   RESERVE_EVE_STORAGE_SIZEGB=$(</proc/cmdline tr ' ' '\012' | sed -ne '/^eve_install_kubevirt_reserve_for_eve_sizeGB=/s#^.*=##p')
-
-   #If not set, default it to 20GB
-   # NOTE reserving 20GB for eve services is an experimental change and could be modified in future after analysing data.
-   if [ -z "$RESERVE_EVE_STORAGE_SIZEGB" ]; then
-      RESERVE_EVE_STORAGE_SIZEGB=20
-   fi
-
-   logmsg "Kubevirt image installing ZFS and EVE reserved storage sizeGB $RESERVE_EVE_STORAGE_SIZEGB"
+   logmsg "Kubevirt image installing ZFS and EVE reserved storage sizeGB"
```

Comment:
Looks like the part after "and" should be removed since there is no more reserved.

Reply:
Done n latest commit